splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -975,7 +975,8 @@
               }
             ]
           }
-        }
+        },
+        "required" : true
```

Comment:
```suggestion
        "required" : {
          "type": "boolean"
        }
```
it should describe that possible value type for field `required` is boolean (true or false).
Reference: https://json-schema.org/learn/getting-started-step-by-step#properties